packages/3id-did-resolver/src/index.ts

```diff
@@ -53,80 +61,128 @@ export function wrapDocument(content: any, did: string): DIDDocument {
   }, startDoc)
 }
 
-const DID_LD_JSON = 'application/did+ld+json'
-const DID_JSON = 'application/did+json'
+function extractMetadata(resolvedState: DocState, fullState: DocState): DIDDocumentMetadata {
```

Comment:
Took me a long time to figure out the difference between "resolvedState" and "fullState".  Can we call them "requestedVersionState" and "currentVersionState"?

And also add a doccomment here?

Reply:
yep!